dubbo-config/dubbo-config-api/src/main/java/com/alibaba/dubbo/config/RefConf.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 1999-2012 Alibaba Group.
+ *  
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *  
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *  
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.alibaba.dubbo.config;
+
+/**
+ * Reference  注解对应的实体配置类
+ * 
+ * @author zhouzhipeng
+ */
+public class RefConf {
```

Comment:
"RefConf.java" does not meet the dubbo code specification, new configuration object should extend existing objects or reuse, see ReferenceConfig and so on. Current writing will break compatibility, eg: adding an attribute, your object cannot support, must modify again.

Reply:
"RefConf.java"   is not a full new configuration class. It is the normal  javabean version for "@Reference"   annotation . Because the "@Reference"   annotation  can not be initialized manual .